data/org.freedesktop.impl.portal.Notification.xml

```diff
@@ -40,7 +40,19 @@
         the notification is replaced by the new one.
 
         The format of the @notification is the same as for
-        org.freedesktop.portal.Notification.AddNotification().
+        org.freedesktop.portal.Notification.AddNotification() plus
+        these private meta-data keys (all optional):
+        <variablelist>
+          <varlistentry>
+            <term>desktop-id s</term>
```

Comment:
Ok, fair enough. I'm also wondering if it would be the case to use `impl-` prefix or something for extra parameters, to make it clear, and maybe just filter them out better if in future we add more stuff here.